Cdn_Plugin.php

```diff
@@ -743,6 +746,33 @@ private function wp_prepare_attachment_for_js_url( $url ) {
 		return $url;
 	}
 
+	/**
+	 * Adjusts attachment image src to cdn. This is for those who rely on
+	 * wp_get_attachment_image_src()
+	 *
+	 * @param 	array   $image	Image object
+	 * @return 	array
+	 */
+	function wp_get_attachment_image_src( $image ) {
+		$url = ( ! emtpy( $image[0] ) ? trim( $image[0] ) : null );
```

Comment:
Fix typo.  Also, it;s better practice to evaluate true instead of not (`!`).  Also, parentheses are not needed around the ternary.
Might as well use: `$url = empty( $image[0] ) ? null : trim( $image[0] );`

Reply:
Updated to your suggestion. Sorry for missing my typo before submitting